packages/compass-components/src/components/combobox/Combobox.tsx

```diff
@@ -245,9 +248,7 @@ export function Combobox<M extends boolean>({
           }
         }
         setSelection(newSelection as SelectValueType<M>);
-        (onChange as onChangeType<true>)?.(
-          newSelection as SelectValueType<true>
-        );
+        (onChange as onChangeType<true>)?.(newSelection);
```

Comment:
I ran --fix on this file to get the type imports thing to go away and it might have gone too far and removed some type assertions that may or may not have been needed. We'll see. I've been bitten by this before.